tox.ini

```diff
@@ -34,20 +34,18 @@ commands =
   coverage: coverage report 
 
 [testenv:lint]
-basepython: python3.7
+basepython: python3.8
 recreate = True
 deps =
   -c dev-requirements.txt
   pylint
   flake8
   isort
   black
+  psutil
 
 commands_pre =
-  pip install {toxinidir}/azure_monitor
+  python scripts/eachdist.py install --editable
```

Comment:
Do we need an `eachdist.py` file? Our packages are not as complicated so there's not really a need to have a separate script to run or CI. Also, I'm not sure you can simply copy the script from OT, maybe there are legal implications?

Reply:
Current bash solution requires contributors to install multiple tools to have it working in Windows, python execution is multi platform friendly, I have no idea about legal implications, we are taking a lot of code from OC and OT even if we wrote it in the first place, so we better understand this better

Reply:
Added bash in the meantime, will follow up about this